### ISSUE TITLE
pavucontrol and GTK-Critical terminal spam

### ISSUE THREAD

**Issue by ghost** [thesquash/gtk-theme-raleigh#2]
when I run pavucontrol from the terminal I get these two messages spammed nonstop: 
![1645299438](https://user-images.githubusercontent.com/89221161/154817479-622f6906-5890-4889-b59e-c109250caf83.png)

It happens as soon as you open it but sometimes it doesn't and you can trigger it by going to the Outputs tab and clicking the mute button. It's very strange.

I'm not sure if it's a *problem* per se but I was just wondering if you were aware of it or if you know what's causing it.

**Comment by thesquash** [comment]
Oops!  I wrote the theme to test for the `.empty` style class on the progress indicator itself, not the containing "trough" the indicator rests in.  Turns out GTK+ (stupidly) assumes that any part of a widget with a border image set should be of a nonzero width -- this assumption is not made for any other type of "decoration" around the widget, that I know of.  `pavucontrol` sets the position of the progress bar every ~1/20 of a second, and if the volume is muted, then the progress bar will always read zero; the indicator will be zero pixels wide, and GTK+ will complain.  Fortunately GTK+ *does* add a flag to a zero-percent-filled progress bar (the `.empty` style class I mentioned before), so now the theme only adds a border image if that style class is **not** present.

To summarize, I made a mistake, sorry for that, and thanks for noticing!
